src/helpers/formatData.js

```diff
@@ -0,0 +1,8 @@
+const formatData = (listsArray, itemsArray) => {
+  return listsArray.map(list => {
```

Comment:
use forEach instead of map?
if you want to use map you need to clone the whole 2 object
so just forEach so it doesnt give off that false feeling of cloning
remember the deep clone and the normal clone from week2